modules/ROOT/pages/endpoint-as2-send.adoc

```diff
@@ -7,44 +7,57 @@ Create the AS2 identifiers for your organization and your trading partners in th
 [%header%autowidth.spread]
 |===
 |Setting |Description |Required
+
+| Owner
+| The trading partner or host that owns the endpoint. The value in this field is populated automatically and cannot be modified.
```

Comment:
Please remove "The value in this field is populated automatically and cannot be modified."
Depending on how the user go to the endpoint create modal, they can modify the owner (when they went to endpoint create via Message flow wizard)

Reply:
Done. 

Also removed this sentence in the SFTP and HTTPs sent topics.